product_docs/docs/biganimal/release/administering_cluster/projects.mdx

```diff
@@ -88,18 +88,19 @@ To delete a project that you created:
 1. Go to **Settings** on the left-side navigation.
 1. From the **Settings** list, select **Security**.
 1. Select **Add a key**. 
-1. On the **Add a key** page, select:
-   - Select the **Provider & Region**
-   - Select **Next**
-   - Enter AWS Key Management System ARN
-   - Enter a friendly name for your key
-   - Select **Finish**
+1. On the **Add a key** page, select the **Cloud Service Provider**. 
+1. Select the **Region** for the key. The interface only displays the regions available in the cloud account you configured.
+   !!! Note 
+   GCP offers the option to configure global keys. If you require a global key, do not select any region.
+   !!! 
+1. Complete the remaining fields according to your cloud provider. 
```

Comment:
I don't think we need provider-specific instructions here, as the UI is very clear. 

The only "difficult" thing is obtaining the key version for the Azure config. We can consider adding a hint/note.